src/components/ApplicationJSon/MenuListPrompt.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div v-if="!isEditing" class="blip-container menu-list-prompt">
+    
+    <div :class="isFailedMessage(status, position)">
+      <div :class="'bubble ' + position">
+        <div v-if="deletable" class="editIco trashIco" @click="trash(document)">
+          <img :src="trashSvg" />
+        </div>
+        <div v-if="editable && !isEditing" class="editIco" @click="toggleEdit">
+          <img :src="editSvg" />
+        </div>
+        <div class="disable-selection">
+          <div class="header-text">
+            <bds-typo variant="fs-16" bold="bold" line-height="plus" class="disable-selection menu-list-prompt-option" v-bind:class="{ readonly: readonly }">{{ headerText }}</bds-typo>
```

Comment:
We should use `v-html="sanitize(text)"`?

Reply:
As we talked about, it's best to stick to the traditional way to prevent html rendering.